primitives/src/rescue/mod.rs

```diff
@@ -881,7 +831,7 @@ mod test_permutation {
 
     fn test_sponge_helper<F: RescueParameter>() {
         let rescue_prp = PRP::default();
-        let rescue_permutation = Permutation::from(rescue_prp.clone());
+        // let rescue_permutation = Permutation::from(rescue_prp.clone());
```

Comment:
Remove this comment.

Reply:
Done in https://github.com/EspressoSystems/jellyfish/pull/146/commits/b2bb4c7d2dfd6f1ed8dd8b3de884374b7e5ddbfd.